skimage/transform/_geometric.py

```diff
@@ -612,6 +612,21 @@ def __call__(self, coords):
         coords_out : (N, D) array
             Destination coordinates.
 
+        Examples
+        --------
+        The call method enables the class instance
+        to behave like functions. The user can call
+        these instances like functions.
```

Comment:
```suggestion
        to behave like a function.
```
Formatting-wise, I don't think lines need to be that short. Content-wise, have you seen the comments by @alexdesiqueira and @mjmdavis?